cli.js

```diff
@@ -68,7 +94,7 @@ const { config } = explorer.load() || {};
 const options = merge({ flags }, config);
 
 if (cli.input.length) {
-  [flags.config] = cli.input;
+  flags.config = cli.input[0];
```

Comment:
why this change? pretty sure that fails linting due to the `prefer-destructuring` rule.

Reply:
Because it is using ES6 features for _nothing_. It is a _clever trick_ but does not improve readability and may be confusing for JS beginners. I've reverted this change.